packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -105,27 +129,33 @@ export class Checkbox extends React.Component<ICheckbox> {
       id,
       selected,
       label,
+      hint,
       value,
       onChange,
       size = 'small'
     } = this.props
     return (
-      <Wrapper>
-        <Input
-          id={id}
-          role="checkbox"
-          checked={selected}
-          type="checkbox"
-          name={name}
-          value={value}
-          onChange={onChange}
-          size={size === 'large' ? 40 : 16}
-        />
-        <Check size={size}>
-          {selected && (size === 'large' ? <TickLarge /> : <Tick />)}
-        </Check>
-        <Label htmlFor={id}>{label}</Label>
-      </Wrapper>
+      <>
+        <Text variant={'reg16'} element={'h2'}>
```

Comment:
Lets rather use "span" or "p" as the hint isn't a title/heading. The h1-h6 element props represents these elements https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Also let's not render the `<Text>` component if hint is not defined:
```
{hint && <Text ...>{hint}</Text>}
```

Reply:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/1322608/204780216-faf39f18-e8b6-44c5-b54d-5cbebfa7a20c.png">
The hint color should be grey500: `<Text ... color="grey500">`

Reply:
The Label should also be optional

Reply:
@jpye-finch Please can you elaborate on the optional label above the hint?